src/components/calculator.js

```diff
@@ -1,58 +1,218 @@
-import './calculator.css';
-import React, { useState } from 'react';
-import calculate from '../logic/calculate';
+/* eslint-disable linebreak-style */
+/* eslint-disable */
```

Comment:
- [ ] `Linters` are tools that perform static analysis on software in order to recognize and report adherence/non-adherence to some set of coding best practice.Instead of disabling `eslint`, it will be nice to fix the issue.